action/claimreward_test.go

```diff
@@ -16,24 +15,6 @@ var (
 	_errNegativeNumberMsg = "negative value"
 )
 
-func TestClaimRewardProto(t *testing.T) {
-	r := require.New(t)
-
-	rc := &ClaimFromRewardingFund{}
-
-	p := rc.Proto()
-	r.NotNil(p)
-	r.IsType(&iotextypes.ClaimFromRewardingFund{}, p)
-	r.Nil(p.Data)
-
-	rc.amount = big.NewInt(100)
-	rc.data = []byte{1}
-	p = rc.Proto()
-	r.NotNil(p)
-	r.EqualValues([]byte{1}, p.Data)
-	r.EqualValues("100", p.Amount)
-}
-
```

Comment:
we can keep this test?

Reply:
`Proto()` related tests already in `rewardmsg_test.go`